views/default/input/month.php

```diff
@@ -0,0 +1,21 @@
+<?php
+/**
+ * Elgg month input
+ * Displays a month input field
+ *
+ * @package Elgg
+ * @subpackage Core
+ *
+ * Note: type="month" is not supported in:
+ * Firefox, Safari or Internet Explorer 11 and earlier versions.
```

Comment:
Please remove notes about non supported browsers if there are no version constraints.
You never know if a future version will add support.